packages/graphql-codegen-cli/src/codegen.ts

```diff
@@ -414,13 +423,13 @@ export async function executeCodegen(
     {
       rendererOptions: {
         clearOutput: false,
-        collapse: true,
+        collapseSubtasks: true,
```

Comment:
`collapse` is renamed to `collapseSubtasks` in [v6](https://github.com/listr2/listr2/releases/tag/v6.0.0)